BlueprintUICommonControls/Sources/AttributedLabel.swift

```diff
@@ -12,6 +12,9 @@ public struct AttributedLabel: Element, Hashable {
     /// the way text is distributed within the line height.
     public var textRectOffset: UIOffset = .zero
 
+    public var isAccessibilityElement = true
```

Comment:
Would be great to add doc comments since these are public (I know we aren't consistent in this repo, but gotta start somewhere)